students/kulon_aleksander/lesson_04/unit_tests_distance.py

```diff
@@ -0,0 +1,35 @@
+import math
+import unittest
+import pytest
+from length_of_the_segment import distance
+
+
+class TestDistanceFunction(unittest.TestCase):
+
+    def test_input_none(self):
+        with pytest.raises(TypeError):
+            distance(None)
```

Comment:
I supposed that this test not working as you designed. You have to pass four parameters to ```distance``` function - you passed only one and that's the reason for the raised TypeError exception. Try to run ```distance(None, 1, 2, 3)```